crates/wasmi/src/table/error.rs

```diff
@@ -38,21 +31,14 @@ pub enum TableError {
         /// The [`TableType`] which is supposed to be a supertype of `ty`.
         other: TableType,
     },
+    TooManyTables,
 }
 
 impl Display for TableError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Self::GrowOutOfBounds {
-                maximum,
-                current,
-                delta,
-            } => {
-                write!(
-                    f,
-                    "tried to grow table with size of {current} and maximum of \
-                    {maximum} by {delta} out of bounds",
-                )
+            Self::GrowOutOfBounds => {
+                write!(f, "out of bounds table growth")
```

Comment:
What is the reason this error information got removed?

Reply:
There's a little bit of awkwardness in one of the call paths that returns these -- basically have to do the calculation of current and maximum in two places -- and I thought given that the `MemoryError::GrowOutOfBounds` case has no such fields, and you expressed dissatisfaction with the error structures in general, I'd just drop the extra fields. But I'm happy to revert it if you want to keep that resolution. I'll push a restoration + calculation change.

Reply:
Okay, well, in all honesty I don't know if we gain too much from the lost information anyways. So do what suits you best. Fixing the error structure can be done in another PR.